packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -280,6 +281,7 @@ class AppDomain extends Domain {
   AppDomain(Daemon daemon) : super(daemon, 'app') {
     registerHandler('start', start);
     registerHandler('restart', restart);
+    registerHandler('callServiceExtension', callServiceExtension);
```

Comment:
It's a typedef used specifically for the daemon protocol's domains to handle json-rpc calls.